### PR TITLE
chore: added default issue template (#96)

### LINKED ISSUE #96: Default issue template
To enable people to leave better issues, we need to use an issue template to help provide the right information

### PR DESCRIPTION
**Pull request by sudiptob2**
Closes #96

## 📑 Description
Added a default issue template for GitHub.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+### Your environment
+* version of Kubernetes
+* which host OS and its version
+
+### Steps to reproduce
+Tell us how to reproduce this issue.
+
+### Expected behaviour
+Tell us what should happen
```
Comment (thschue) [inline .github/ISSUE_TEMPLATE.md:12]:
```suggestion
<!-- Tell us what should happen -->
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### Actual behaviour
+Tell us what happens instead
```
Comment (thschue) [inline .github/ISSUE_TEMPLATE.md:15]:
```suggestion
<!-- Tell us what happens instead -->
```

**Review by thschue — CHANGES_REQUESTED** [review]
Hello @sudiptob2! Thanks for your contribution. Made some suggestions.

**Comment by thschue** [comment]
You could also add an additional information section, like we did here: 

https://raw.githubusercontent.com/k8sgpt-ai/k8sgpt/main/.github/pull_request_template.md

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+### Subject of the issue
+Describe your issue here.
+
+### Your environment
+* version of Kubernetes
+* which host OS and its version
+
+### Steps to reproduce
+Tell us how to reproduce this issue.
```
[the remark this answers is not in the document]
Comment (sudiptob2) [reply .github/ISSUE_TEMPLATE.md]: updated

**Review by thschue — APPROVED** [review]
I like it! LGTM!

**Pull request merged**